filer/static/filer/js/addons/tooltip.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+/* global django */
+
+var filerTooltip = function ($) {
```

Comment:
i don't really like this being a global
do we have a filer namespace? it would be good to scope everything we do under it